pipelines/tasks/galasabld-command.yaml

```diff
@@ -21,10 +21,13 @@ spec:
   - name: galasabldImageTag
     type: string
     default: main
+  - name: image
+    type: string
+    defualt: harbor.galasa.dev/galasadev/galasabld-amd64:$(params.galasabldImageTag)
```

Comment:
I don't think it's possible to reference a parameter from inside a parameter, so can we change this to just the part before the colon?